addons/auth_passkeys/models/passkey.py

```diff
@@ -0,0 +1,32 @@
+from odoo import fields, models, _
+
+class PassKey(models.Model):
+    _name = "auth.passkeys.key"
+    _description = "Passkeys"
+    _order = "id desc"
+
+    name = fields.Char(required=True)
+    credential_identifier = fields.Char(required=True)
+    public_key = fields.Char(required=True)
+    sign_count = fields.Integer(default=0)
+
+    def _get_nopw_user_by_id(self, id):
+        # We strip the = because webauthn base64 padding behaves a little different depending on the platform (win/yubi)
+        results = self.sudo().search([("credential_identifier", "like", id.strip("=")+"%")])
+        if len(results) == 1:
```

Comment:
Shouldn't we set a `UNIQUE` SQL constraint on `credential_identifier`? Then you could replace this `if` by a `limit=1` in the search call above.